app/assets/stylesheets/darkswarm/shop_tabs.css.scss

```diff
@@ -76,20 +76,17 @@
   // content revealed in accordion
 
   .page-view {
-    margin-bottom: 5em;
     background: none;
+    background-color: $grey-080;
     border: none;
+    padding-bottom: 5em;
```

Comment:
the upper part of the background below this image was margin (doesnt take the background color of the content and we cannot change background color of margin) now it's padding (takes the background color of the content)
![image](https://user-images.githubusercontent.com/1640378/79683595-2980e200-8223-11ea-891d-e96b59878d15.png)